intro.tex

```diff
@@ -1,55 +1,67 @@
 \section{Introduction}
 
-This  note describes the plan for ensuring the Rubin community will have the data products and services necessary to produce early science during the time between commissioning and the first data release in year 1. 
-The current planned start date for full survey operations is 1 April 2024, with considerable uncertainty. 
-As of November 2021, the construction project schedule shows completion between December 2023 and July 2024 (i.e. planned finish and finish with contingency). 
-It is unlikely the project will finish in December 2023 and highly likely it will finish by July 2024. 
+This note describes the plan for ensuring the Rubin community will have the data products and services necessary to produce high-impact early science during the time between commissioning and the first LSST Data Release, DR1.  
+Community expectations for early science are high due to the transformative nature of the Rubin data and the extensive amount of on-sky time planned in commissioning and science validation.
 
 \subsection{Definition of Early Science}  \label{ssec:defn}
 
 Early Science (ES) is defined as any science enabled by Rubin for its community through and including the first data release, \drone.
-DR1 will be based on the first 6 months of data and is scheduled for release 12 months  after the start of full survey operations, \citep{RDO-011}.
-Community expectations for early science are high due to the transformative nature of the Rubin data and the extensive amount of on-sky time planned in commissioning and science validation.
 
-\subsection{Motivations for an Early Science Program}
+DR1 is the first planned release of static-sky data products. 
+It will be based on the first 6 months of data and released 12 months after the start of full survey operations.  
+Alerts to transient, variable and moving objects are produced by the LSST Prompt Processing pipelines via Difference Image Analysis (DIA), which creates difference images by subtracting a template image \footnote{Templates are transient-free coadds produced by stacking previous images such as to remove the flux of any transient or moving object.} from each new image acquired, and then identifying sources in the difference image. 
+Without templates for a given pointing and filter, there can be no Alerts.
+Templates are produced during Data Release Processing (DRP) and initially released as part of DR1, so in year 1 there will be no templates from which to generate alerts, and hence no Alerts.
```

Comment:
Strongly agree with Eric's point regarding conditional wording. 

It might be even better to significantly modify the sentence to avoid confusion, for example,
"Templates are normally produced during Data Release Processing (DRP) and would initially be released as part of DR1, so separate processing campaigns would be needed to produce templates and generate alerts prior to DR1."

Reply:
I worry that this would set an expectation that that we would put a dedicated processing campaign in place to produce templates.

Reply:
We _will_ do some kind of dedicated processing to make incremental templates, though.  Whether or not you call that a "campaign" depends on your definition, but the images won't coadd themselves :-)